package.json

```diff
@@ -29,17 +28,17 @@
 		"dev": "cross-env NODE_ENV=production nx run storybook:build:docs && nx start docs",
 		"format": "yarn formatter tag:component",
 		"formatter": "nx run-many --target format --projects",
-		"preinstall": "command -v nvm >/dev/null 2>&1 && nvm use || exit 0",
```

Comment:
https://yarnpkg.com/advanced/lifecycle-scripts

`preinstall` tasks are deprecated and docs suggest migrating to `postinstall`.